connection.go

```diff
@@ -518,15 +518,18 @@ func (c *Connection) Reply(ctx context.Context, opErr error) error {
 		if opErr == nil {
 			c.debugLog(fuseID, 1, "-> %s", describeResponse(op))
 		} else {
-			c.debugLog(fuseID, 1, "-> Error: %q", opErr.Error())
+			if c.shouldLogError(op, opErr) {
```

Comment:
Now that you moved the error logging code into this branch of the conditional, that means that users will not see error logs when they don’t enable debug logging, which does not seem right to me.

It might be easier to split the error/debug log reduction into a separate PR to unblock adding the operation ID.

Reply:
Raised separate PR - https://github.com/jacobsa/fuse/pull/172